include/common_package/commands.hpp

```diff
@@ -0,0 +1,421 @@
+#pragma once
```

Comment:
Do we have any guideline on when it is okay to add function logic to .hpp files? I think in this case (a common library) it would be fine. What do you think @J0las?